ngraph/core/include/ngraph/function.hpp

```diff
@@ -52,6 +53,11 @@ namespace ngraph
                  const ParameterVector& parameters,
                  const std::string& name = "");
 
+        Function(const ResultVector& results,
```

Comment:
ResultVector or OutputVector? - @ilyachur @GlebKazantaev please look at this

Reply:
OutputVector is a helper. We can add two constructors in order to be consistent with previous approach.

Reply:
Also there are constructors with NodeVector and shared_ptr<Node> as result . Do we need to copy all of them and add sinks there?

Reply:
I guess these (Results, Sinks, Parameters) and (Outputs, Sinks, Parameters) will be enough.

Reply:
done